### PR TITLE
React UI

### PR DESCRIPTION
**Pull request by David-Prism**
Basic boilerplate react UI.

### CODE REVIEW

File: functions/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions",
+    "ms-python.python"
+  ]
+}
```
Comment (theKunte) [inline functions/.vscode/extensions.json:6]: We might need to remove all of the .vsode files from our folder. I think in toddy's lecture he recommended to move these files into the .gitignor file. 

PS: Jak create the entire file structure for our team

**Review by farhadbahrehmandhenry — COMMENTED** [review]
as you can see there are a lot of changed files in your branch that is not related to the feature you added. if we merge this branch we actually have merged some unfinished features. I don't know if it's ok or not.
also you probably need to add a documentation to your PR too explaining how to run the app you created.

**Comment by toddysm** [comment]
As @theKunte and @farhadbahrehmandhenry noted above, there are a lot of files that should not be committed to the repo. Also, avoid cluttering the repo with `test` files.

**Comment by David-Prism** [comment]
## To run the app, run the commands:
-Navigate to the ui folder, then the react-ui folder
-Run `npm install`
-Run `npm start`

**Review by farhadbahrehmandhenry — APPROVED** [review]

**Comment by theKunte** [comment]
npm install and npm start works and I was able to run it on my machine.

**Review by theKunte — APPROVED** [review]

**Review by theKunte — CHANGES_REQUESTED** [review]
Actually, I was not able to run the react app

**Review by theKunte — APPROVED** [review]
Never mind :D 
I was able to run the app after navigating to the react-ui folder
image

**Pull request closed without merging**